scheduler/src/main/kotlin/net/plan99/graviton/scheduler/LinuxTaskScheduler.kt

```diff
@@ -0,0 +1,67 @@
+package net.plan99.graviton.scheduler
+
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.time.Duration
+
+/**
+ *
+ * @author Anindya Chatterjee
+ */
+class LinuxTaskScheduler : OSTaskScheduler() {
+
+    override fun register(taskName: String, task: OSScheduledTaskDefinition) {
+        require(Files.isRegularFile(task.executePath)) { "Cannot access ${task.executePath}" }
+        val cronExpression = task.frequency.toCronExpression()
+        execute("/bin/bash", "-c", "(crontab -l ; echo \"$cronExpression ${task.executePath}  #$taskName\") 2>&1 | grep -v \"no crontab\" | sort | uniq | crontab -")
```

Comment:
This will have the effect of sorting the users crontab - I wonder if that matters. I do understand why it's done this way. And it probably doesn't matter. I just wonder if it'll annoy anyone who has a big crontab they like the layout of.

Reply:
Yes you are right. `sort` messed up the ordering, never tested with multiple entries. I'll send a correction.